### ISSUE TITLE
Documentation created for the node sdk

### ISSUE THREAD

**Issue by ratnakar-asara** [hyperledger-archives/fabric#1835]
The documentation needs cleaning up.
1.  Do not have to display it locally
   file:///home/barry/src/github.com/hyperledger/fabric/sdk/node/doc/globals.html
2.  The three high globals - hlc, crypto, stats.   These need several things,
   a) pruned to delete all methods, interfaces, variables that are private
   b) provide an example for each function and provide a description.   For example in the Readme.md there is the info for HLC objects and reference documentation.  This would be great if it were tied directly to the api call. 

In many cases we aren't really sure what the call does or which calls are private.

**Comment by smithbk** [comment]
I agree that work needs to be done here.  You can click on upper right to view only public which will help some, but there is still a lot of noise.

**Comment by elli-androulaki** [comment]
May you provide more info on what this refer to? file:///home/barry/src/github.com/hyperledger/fabric/sdk/node/doc/globals.html
Thanks.

**Comment by ratnakar-asara** [comment]
Also need a small update in [troublingshooting](https://github.com/hyperledger/fabric/tree/65c8e77a67490a3a88bca7aba76b2bb83824c527/sdk/node#troublingshooting) section of the document for the below error.
I see point 2 already covers the same,  however, It would be good to add the below error as well.

```
/usr/local/lib/node_modules/hlc/bin/main.js:88
     if (err) throw Error("failed to register and enroll "+userName+": "+err);
                                                                        ^
Error: failed to register and enroll user.4-1466002040123: Error: sql: no rows in result set
```

**Comment by bmos299** [comment]
@smithbk  Also, with the the npm install -g hlc you need to make sure you have the latest level of node.  I know for sure the install works with node v6.2.1

**Comment by srderson** [comment]
I wish we wouldn't continue to use chaincode_example02 in documentation. That chaincode is poorly named and poorly documented. How about using https://github.com/hyperledger/fabric/blob/master/examples/chaincode/go/map/map.go instead?

**Comment by vmorris** [comment]
@ratnakar-asara What solution are you suggesting for this sort of error? I just ran across it myself fooling around with cp-web:

```
Failed to submit chaincode deploy transaction ---> function: init, args:  :  EventTransactionError {
  error: 
   { Error: sql: no rows in result set
       at /home/vance/git/cp-web-new/cp-web/node_modules/hlc/node_modules/grpc/src/node/src/client.js:411:17 code: 2, metadata: Metadata { _internal_repr: {} } },
  msg: 'Error: sql: no rows in result set' }
```

Edit: Never mind, the solution is to delete /tmp/keyValStore
